src/core/chain.ts

```diff
@@ -294,7 +294,7 @@
             chainsByLayer[layer].sort((a: IChain, b: IChain) => { return b.pathLength - a.pathLength });
 
             if (opts.contain) {
-                var containedChains = getContainment(chainsByLayer[layer], opts.contain);
+                var containedChains = getContainment(chainsByLayer[layer], {alernateWindings:true});
```

Comment:
This `{alternateWindings:true}` wouldn't allow someone to pass in their own options in `opts.contains`, for example, if they passed in `false`.
Also there's a typo: `alernate...`  ;-)